capi/src/init-node/html-to-hyperscript.ts

```diff
@@ -40,10 +41,28 @@ const Hyperscript = (
   if (node.nodeName === "#text") {
     return node.value as string;
   }
+
+  if (node.nodeName === "#comment") {
+    return null;
+  }
+
+  if (node.tagName) {
+    if (!validTags[node.tagName] && !node.tagName.startsWith("amplify-")) {
+      throw new Error(
+        `Invalid tag "${node.tagName}" encountered in "${srcPath}"`,
```

Comment:
Can we add how a user could resolve this issue. Maybe just a note about escaping it.